dashboard/app/main.go

```diff
@@ -85,6 +88,12 @@ func initHTTPHandlers() {
 	http.HandleFunc("/cron/subsystem_reports", handleSubsystemReports)
 }
 
+func handleMovedPermanently(dest string) http.HandlerFunc {
```

Comment:
Does much already depend on the old URLs?
If not, I'd just switch to the new ones.

Reply:
I know a few docs using it. Both versions are needed temporarily to safely change them.
And I'd like to share the coverage.json.gz link to follow the new api from the beginning.